code/controllers/master_controller.dm

```diff
@@ -225,6 +225,15 @@ datum/controller/game_controller/proc/cachedamageicons()
 	else
 		log_startup_progress("Not generating minimaps - SKIP_MINIMAP_GENERATION found in config/config.txt")
 
+	if(!config.skip_vault_generation)
+		spawn()
```

Comment:
Due to this `spawn()` the `Finished initializations` message will come up before this even starts to generate the structures.